sync.sh

```diff
@@ -15,6 +7,7 @@ do
   echo "--- $slug" >> module-ids
```

Comment:
Do we need to parametrize this script so `neb` can be invoked to pull content from production / staging / qa based upon the pipeline's needs?